flow/shared/constants.go

```diff
@@ -34,8 +34,8 @@ type (
 )
 
 const (
-	NoopSignal CDCFlowSignal = iota
-	ShutdownSignal
+	NoopSignal     CDCFlowSignal = iota
+	ReservedSignal               // previously ShutdownSignal
```

Comment:
Not sure best way to do this, ideally it'd be removed but then it messes up things for existing paused mirrors